File: src/homepod-mini/device.ts
import HomePodBaseDevice from '../homepod-base/device';
import type HomePodMiniDriver from './driver';

export default class HomePodMiniDevice extends HomePodBaseDevice<HomePodMiniDriver> {
    get discoveryStrategies(): string[] {
        return ['homepod-mini'];
    }
}

File: src/homepod_mini/device.py
from __future__ import annotations

from ..homepod_base.device import HomePodBaseDevice


class HomePodMiniDevice(HomePodBaseDevice):
    """Homey device representing a paired HomePod Mini."""

    @property
    def _device_type_name(self) -> str:
        return 'HomePod Mini'


homey_export = HomePodMiniDevice
